includes/pipeline_snapshot.js

```diff
@@ -0,0 +1,60 @@
+module.exports = (version, params) => {
+    if ((!params.enableMonitoring) || dataform.projectConfig.schemaSuffix) {
+        /* Don't send pipeline snapshot if monitoring is disabled (enableMonitoring is true by default)
+        or if we're in a Dataform development workspace (schemaSuffix is null if we're in a development workspace) */
+        return true;
+    }
+    return operate("pipeline_snapshot_" + params.eventSourceName, ctx => [`

```

Comment:
Operations are SQL scripts and don't create tables, so can't have metadata. This one writes into a table in another project which does have metadata, but it lives there and not here because this operation is not responsible for the schema of the table.

Reply:
Ah that makes sense thanks